### PR TITLE
Allowing `@types/node` v22 (and upgrading it internally)

### PR DESCRIPTION
**Pull request by RobinTail**
`@types/node` version 22 released.

Need this to deduplicate the installation
https://github.com/RobinTail/express-zod-api/pull/1947

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -6,11 +6,11 @@
   "packages": {
     "": {
       "name": "node-mocks-http",
-      "version": "1.14.1",
+      "version": "1.15.0",
```
Comment (RobinTail) [inline package-lock.json:9]: this was out of sync in `master`

**Comment by RobinTail** [comment]
@eugef , please review

File: package.json [hunk 2]
```diff
@@ -47,7 +47,7 @@
   },
   "dependencies": {
     "@types/express": "^4.17.21",
-    "@types/node": "^20.10.6",
+    "@types/node": "^20.10.6 || ^22.0.0",
```
Comment (RobinTail) [inline package.json:50]: Or can do just `*`, like other packages do:

https://www.npmjs.com/package/@types/busboy?activeTab=code

image

File: package-lock.json [hunk 3]
```diff
@@ -6,11 +6,11 @@
...
       "license": "MIT",
       "dependencies": {
         "@types/express": "^4.17.21",
-        "@types/node": "^20.10.6",
+        "@types/node": "^20.10.6 || ^22.0.0",
```
Comment (eugef) [inline package-lock.json:13]: @RobinTail let's go for

```
"@types/node": "*" 
```

Comment (RobinTail) [reply package-lock.json:13]: ok...

Comment (RobinTail) [reply package-lock.json]: done

File: package.json (hunk 2 above)
Comment (RobinTail) [reply package.json]: agreed on star

**Comment by RobinTail** [comment]
Made it a star, @eugef . Please review again.

**Review by eugef — APPROVED** [review]

**Comment by RobinTail** [comment]
@eugef , please publish

**Comment by eugef** [comment]
Version 1.15.1  is published

**Pull request merged**